### ISSUE TITLE
How change default saturation to maximal value to correct working of "hue" slider

### ISSUE THREAD

**Issue by NDanilov2015** [linx4200/vue-color#232]
Hi, Xiaokaike!

_Bug or "feature" in Sketch vue component from vue-color package version 2.8.1, last updating._

if you choose a color firstly from the "rainbow spectrum band" ("HUE slider / color tone"), without touching "buttons with colors" or "big area sector ("Saturation/Brightness")", then I get always black color, regardless of the chosen color tone in reality. Buttons and a "wide rectangle with gradations of brightness / saturation", however, selected the color correctly.

How I can setup default saturation/brightness to max values, without needs touching "big rectangle area" where is "pointer of saturation"? To I can to immediately select a color tone using the slider, without setting the saturation and brightness?

![picture of bug](https://i.ibb.co/dgNz7bF/hsv.png)

**Comment by NDanilov2015** [comment]
Hi again! Can please someone help me with this bug/feature?

**Comment by chenkai0520** [comment]
@NDanilov2015 
You can set an initial color, when you choose a color firstly from the "rainbow spectrum band"  the alpha-value will node change

**Comment by NDanilov2015** [comment]
@chenkai0520 what did you mean? I need preset maximal value saturation/brightness before any manual choosing, strictly programmatically.

**Comment by chenkai0520** [comment]
@NDanilov2015 Yes it is，for example you can set a default color like `#FFFFFF`
